draft-ietf-ppm-dap.md

```diff
@@ -1479,36 +1487,58 @@ struct {
 } InputShareAad;
 ~~~
 
+~~~ tls-presentation
+struct {
+ReportID id;
+ReportError error;
+} ReportUploadStatus;
+
+struct {
+ReportUploadStatus status<0..2^32-1>;
+} UploadResponse;
+~~~
+
 If the upload request is malformed, the Leader aborts with error
 `invalidMessage`.
 
 If the Leader does not recognize the task ID, then it aborts with error
 `unrecognizedTask`.
 
+Otherwise, the Leader responds with HTTP status 200, a body consisting
+of `UploadResponse` and the media type `application/dap-report-resp`. If
+the entire upload has been successfully completed, the Leader MAY respond with
+an empty body.
+
+For each report in the upload, the Leader creates `ReportUploadStatus` struct
+and adds the `ReportId` to it. If there was no error during the processing,
+the Leader sets the `error` to `none`.
```

Comment:
Thanks for implementing this suggestion here @akoshelev. However, after thinking this more, I think this is may not a good idea. The problem I see is that during the aggregation flow, a `ReportError error = none` is not a valid response. If there was really no report error, then the aggregator would expect a different response than a `ReportError`.

That said, I think you and @tgeoghegan have already suggested a much better idea (https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/686/files#r2070551993): if a report was uploaded successfully, then simply don't include it in the `UploadResponse`.